### DISCUSSION TITLE
my CSS file don't work in GitHub pages

### DISCUSSION THREAD

**Discussion by ahmadkaddah** [community/community#42838] [New to GitHub]
### Select Topic Area

Question

### Body

Hello, everyone

I’m trying to move my Project in  HTML and CSS and JS over here.
HTML is working, but CSS is not being applied
" NOTE: in my visual studio code everything is good but in GitHub, the  CSS file doesn't work

this is the repositories:  https://github.com/ahmadkaddah/Youtuby-Hsoub

**Comment by Shariar-Hasan** [comment]
well, 
I can't see any problem here now, as I saw your webpage.
is there anything else you are facing problem with.

**Comment by souvik666** [comment]
@ahmadkaddah can you try once with adding . like this while importing css  `<link rel="stylesheet" href="./css/style.css"> `
